src/utils/søknad.ts

```diff
@@ -29,7 +54,7 @@ export const fraStringTilTall = (tallAvTypenStreng: string) => {
   }
   return parsed;
 };
-
+// TODO: Ikke i bruk?
```

Comment:
Ikke i bruk? `verdiTilTekstsvar` brukes

Reply:
Nja, det finnes to `verdiTilTekstsvar` ? Den i `visning` brukes iallefall, men finner ikke at denne i `søknad` brukes? Eller er jeg og IntelliJ blinde? :D

Reply:
Oi, du har rett! Jeg glemte nok å fjerne den da jeg flyttet funksjonen. Da er det den i visning som er den riktige, så denne kan fjernes :)